helpers/helpers.go

```diff
@@ -198,10 +198,11 @@ func UnpackFile(file string, dest string) error {
 	return nil
 }
 
-// CopyFile is used during the build process to copy a given file to the target
-// instead of dealing with the particulars of hardlinking.
+// CopyFile copies a file, overwriting the destination if it exists. It is used
+// during the build process to copy a given file to the target instead of
+// dealing with the particulars of hardlinking.
 func CopyFile(dest string, src string) error {
-	return copyFileWithFlags(dest, src, os.O_RDWR|os.O_CREATE)
+	return copyFileWithFlags(dest, src, os.O_RDWR|os.O_CREATE|os.O_TRUNC)
```

Comment:
Makes me wonder if there are other users of copyFileWithFlags which are not doing the right thing, but the one other use (CopyFileNoOverwrite) looks fine.

Good catch.